ci/docker/runtime_functions.sh

```diff
@@ -307,6 +307,7 @@ unittest_ubuntu_python2_cpu() {
     export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
     nosetests-2.7 --verbose tests/python/unittest
     nosetests-2.7 --verbose tests/python/train
+    nosetests-2.7 --verbose tests/python/quantization
```

Comment:
Is it intended to run them as part of unittests? They run on C5 and G3 instances and would fail under the current configuration, otherwise there would be no reason to create a separate job, right?

Reply:
The tests run here are for basic tensor operators such as quantize, dequantize, and requantize, which have both CPU and GPU versions implemented. The operators that can only run on P3 are NN operators such as FC, Convolution, and Pooling.

Reply:
So what happens if an NN operator test is being hit on a G3 instance?

Reply:
NN operators will not run on G3. There is a context check in the python function and it will skip the context of gpu.